routes/articles.js

```diff
@@ -10,4 +10,15 @@ router.get('/', (req, res) => {
     })
 })
 
+router.post('/article', (req,res) => {
+    const db =req.app.get('db')
+    const title = req.body.title
+    const body = req.body.body
+    const author = req.body.author
```

Comment:
You should be able to this instead to do  the same thing:
```
const { title, body, author } = req.body
```

Reply:
Tried, got errors